mkdocs.yml

```diff
@@ -83,4 +83,4 @@ extra:
         link: https://k0sproject.io/
         name: Lens Website
```

Comment:
Maybe fix this too while poking at this file :)

Reply:
This was fixed. PR is ready for a review.